inc/commonitilobject.class.php

```diff
@@ -1745,6 +1760,7 @@ function post_addItem() {
 
       // Add document if needed, without notification
       $this->input = $this->addFiles($this->input, ['force_update' => true]);
+      $this->input = $this->addFiles($this->input, ['name' => 'content', 'force_update' => true]);
```

Comment:
Can you explain why there is 2 calls to addFiles here (in a code comment) ?

Reply:
There are 2 calls because the file area in a ticket handles the uploads dropped in its own area, but also screenshots dropped in the textarea. The textarea is named "content", and the file area is named "filename". 

The first call handles the uploads found in $this->input['_filename'] 

The second call handles files uploads (aka screenshots) found in this->['_content']

Reply:
Names seems confusing to me

Reply:
I agree with you, that's whiy I'm cautious with this PR.

However, I believe I can continue the PR to completely cut the functional link between the file input and the textarea. This would slightly change the UI, and solve this mess.